src/uu/chmod/src/chmod.rs

```diff
@@ -206,13 +207,23 @@ impl Chmoder {
                         ));
                     }
                 } else if !self.quiet {
-                    return Err(USimpleError::new(
-                        1,
-                        format!(
-                            "cannot access {}: No such file or directory",
+                    if files.len() == 1 {
+                        return Err(USimpleError::new(
+                            1,
+                            format!(
+                                "cannot access {}: No such file or directory",
+                                filename.quote()
+                            ),
+                        ));
+                    } else {
```

Comment:
I don't think we need this distinction if we `continue` anyway. You can get both cases with something like this:
```rust
show!(USimpleError::new(
    1,
    "cannot access {}: No such file or directory",
    format!(
        "cannot access {}: No such file or directory",
        filename.quote()
    ),
));
continue;
```

Reply:
Oh yeah, of course :see_no_evil: thanks for pointing it out!